### ISSUE TITLE
The distance is not counted correctly while moving diagonally

### ISSUE THREAD

**Issue by droideck** [Saibot393/enhancedcombathud-pf2e#7]
The distance counter (which shows squares on the right side of the portrait) counts the squares just 1 by 1. But in pf2e, the first square of diagonal movement you make in a turn counts as 5 feet, but the second counts as 10 feet, and your count thereafter alternates between the two.

Could you please check this when you have time? Otherwise, it's an awesome tool and a great pf2e implementation! Thank you so much for the work!

**Comment by Saibot393** [comment]
I will look into it, should not be too hard to implement

**Comment by droideck** [comment]
Just a short update for v1.6.0.
When I grab the token and move it diagonally, three squares with one move, it's all good, and the distance is counted correctly.

But when I do move through the squares one by one - it doesn't work, and the UI doesn't remember that I already moved diagonally. Hence, it counts each square separately when moving one by one.

**Comment by Saibot393** [comment]
Yeah, couting diagonall moves is a bit harder then initially thought. I am still looking at a good solution for it

**Comment by KBako** [comment]
Hi! I was wondering if there is any update on this fix? As far as I can tell, the original movement is still not accounting for the movement setting in Configure Settings/Core/ Grid Diagonals.

If there is a way to get the step tracker accurate to the setting at this location? Thanks!

**Comment by Saibot393** [comment]
There is not yet, sorry.
